### ISSUE TITLE
missing author information from jsonfeed

### ISSUE THREAD

**Issue by jackjamieson2** [jackjamieson2/yarns-microsub-server#80]
e.g. when I parse manton.org/feed.json, the feed author information is not added to items

Looks like this is because the author information is not stored as an h-card like with other feeds

It returns with the author information as properties at the beginning of the feed. 
```
{
  "type": "jsonfeed",
  "name": "Manton Reece",
  "url": "https:\\/\\/www.manton.org\\/feed.json",
  "photo": "https:\\/\\/micro.blog\\/manton\\/avatar.jpg",
  "items": [
...
}
```

Yarns is expecting an author property as an h-card
```
"author":{
  "name": "Manton Reece",
  "url": "https:\\/\\/www.manton.org\\/feed.json",
  "photo": "https:\\/\\/micro.blog\\/manton\\/avatar.jpg",
}
```

Two ways to address this:

1) I can add a bit to Yarns to convert this

2) This could be addressed in Parse-This -- @dshanske, what do you think?  I'm happy to handle it in Yarns but I expect this would be useful in other cases of Parse-This as well.

I'm going to implement it in Yarns itself in the meantime since it's pretty simple and I want jsonfeeds to work asap

**Comment by jackjamieson2** [comment]
Similarly, items parsed from jsonfeeds are missing a type='entry' property

**Comment by dshanske** [comment]
@jackjamieson2 Will fix this. It should be on the library side

**Comment by jackjamieson2** [comment]
Great, I implemented a quick fix to the dev branch of yarns but I agree it
should be in the library

**Comment by dshanske** [comment]
@jackjamieson2 If you look at the JSONfeed you cited, and all micro.blog jsonfeeds, no author property exists. I'll fix the other problem.

**Comment by dshanske** [comment]
Not every feed has an author provided. Not sure what Microsub expects if there isn't one provided.

**Comment by jackjamieson2** [comment]
Ah, I thought that might have been a jsonfeed convention, but I didn't check the spec. 

Nonetheless, my inclination would be to interpret  the feed `name` `url` and `photo` properties as belonging to the author. At least for Micro.blog feeds since that seems to be the typical use.  Without that, jsonfeeds from Micro.blog are rendered in Monocle (and presumably other clients, though I didn't check yet) as having no author.

Beyond that, it looks like there's discussion of using a `_source` property to handle feed names etc. apart from authors https://github.com/indieweb/microsub/issues/13

I'm heading to sleep now so I'll check properly in the morning, but it looks like this is implemented in Monocle (in `views/timeline/author-card.php`)

**Comment by jackjamieson2** [comment]
Just tried adding the feed `name` etc. to `_source` - It does not display in Monocle. (Looking at Monocle's source).

<img width="590" alt="Screen Shot 2019-06-20 at 11 23 05 AM" src="https://user-images.githubusercontent.com/12275399/59861041-158ff980-934e-11e9-8434-2dd94da20de1.png">

<img width="641" alt="Screen Shot 2019-06-20 at 11 23 20 AM" src="https://user-images.githubusercontent.com/12275399/59861047-17f25380-934e-11e9-8fb4-0583019a6342.png">

**Comment by dshanske** [comment]
I would rather start providing a source property for everything than the alternative

**Comment by jackjamieson2** [comment]
Makes sense. I think I’ll keep this workaround in Yarns temporarily, and
remove it if/when _source can be used more reliably.
